modules/sql-engine/src/integrationTest/sql/types/char/test_char_character_set.test

```diff
@@ -0,0 +1,60 @@
+# name: test/sql/types/char/test_char_character_set.test
+# description: E021-01 Character string types. CHARACTER SET tests
+# group: [char]
+
+# Incorrect name of charset
+statement error
+CREATE TABLE t_incorrect(C1 CHAR CHARACTER SET UTF_8);
+
+statement ok
+CREATE TABLE t_default (c1 CHAR, c2 CHARACTER, c3 CHAR(3), c4 CHARACTER(3));
+
+statement ok
+CREATE TABLE t_utf8 (c1 CHAR CHARACTER SET UTF8, c2 CHARACTER CHARACTER SET UTF8, c3 CHAR(3) CHARACTER SET UTF8, c4 CHARACTER(3) CHARACTER SET UTF8);
+
+statement ok
+CREATE TABLE t_latin1 (c1 CHAR CHARACTER SET LATIN1, c2 CHARACTER CHARACTER SET LATIN1, c3 CHAR(3) CHARACTER SET LATIN1, c4 CHARACTER(3) CHARACTER SET LATIN1);
+
+statement ok
+INSERT INTO t_default VALUES('1', '2', '🍞', 'Ю44');
+
+statement ok
+INSERT INTO t_utf8 VALUES('1', '2', '🍞', 'Ю44');
+
+# Must fail due to use symbol doesn't belong to charset of column
+skipif ignite3
+# https://issues.apache.org/jira/browse/IGNITE-22100
+statement error
+INSERT INTO t_latin1 VALUES('1', '2', '🍞', 'Ю44');
```

Comment:
seems we also need to check equality of returning types for char char(1) and character(1) ?
SELECT TYPEOF(SELECT '1'::char), TYPEOF(SELECT '1'::char(1))

Reply:
`typeof` it's not a standard function and yes, our implementation returns type together with length, but I think it's not require to check it here.

Reply:
yes i understand that this is not standard function i mean use this func to check corresponding equality

Reply:
Do you have an example in real life when it will be useful?